### PR TITLE
Send indv. events to HEC, add timeout, add new tests

### PR DESCRIPTION
**Pull request by OllieJC**
- Instead of making a big payload, send each event
- Add a default 10 second timeout to the POST request to Splunk (available as env. variable)
- New tests include a multi event fixture and another time format.

### CODE REVIEW

**Comment by OllieJC** [comment]
@emisaur / @chrisfarms - these are some shonky changes to the shonky Lambda to hopefully make it a bit _better_.
Another thing not mentioned above is that we're also now dropping `/healthcheck` from the `kube-probe`.
If there is not anything glaringly stupid with these changes, we can merge and build a ZIP to get deployed to eIDAS.

File: fluentdhec/lambda_function.py [hunk 1]
```diff
@@ -83,6 +102,7 @@ def build_payload_k8s(data, context):
             time = extract_time(jlog['log'])
             if time:
                 event["time"] = time
+                print(time)
```
Comment (OllieJC) [inline fluentdhec/lambda_function.py]: Shouldn't be here.

File: tests/test_lambda_handler.py [hunk 2]
```diff
@@ -79,6 +86,12 @@ def test_syslog_3_extract_time():
     assert resp == 869077200
 
 
+def test_syslog_4_extract_time():
+    message = "syslog INFO [2019/May/20:04:01:30 PM] temp"
+    resp = fluentdhec.lambda_function.extract_time(message)
+    assert resp == 1558364490
+
+
```
Comment (emisaur) [inline tests/test_lambda_handler.py]: Little confused as to why there's two functions with the same name here?

Comment (OllieJC) [reply tests/test_lambda_handler.py]: Good catch, they are no longer the same name :D

**Pull request merged**